index.js

```diff
@@ -96,33 +98,43 @@ if (process.env.LEO_LOGGER) {
 	if (process.env.LEO_LOGGER == "true") {
 		module.exports.configure(true);
 	} else {
-		let parts = process.env.LEO_LOGGER.split("/");
-		let regex;
-		let config;
-		let lookup = {
-			a: "all",
-			t: "time",
-			i: "info",
-			d: "debug",
-			e: "error",
-			s: "info",
-			v: "debug",
-			T: "printTimestamp"
-		};
-		if (parts.length == 1) {
-			regex = new RegExp(parts[0]);
-		} else if (parts.length > 2) {
-			regex = new RegExp(parts[1]);
-			let flags = parts[2];
-			config = {
-				all: flags.length == 0
+		let loggers = process.env.LEO_LOGGER.split(";");
+		loggers.forEach(logger => {
```

Comment:
@jgrantr I tested this different ways but wanted second eyes.  It does the same logic as before but just for each split of ';'